CHANGELOG.md

```diff
@@ -37,7 +37,10 @@
 
 ##### Enhancements
 
-* None.  
+* Add `:headers` option to allow passing in custom headers to `cURL` when downloading source via the `:http` download strategy.
```

Comment:
sorry one more nit: please add two empty spaces at the end of this line for markdown formatting.

Reply:
this is still in the wrong place since 1.7.4 was shipped lets move it up again to `master` section

Reply:
O geez, how did I manage that. Fixed.